src/System.Globalization.Calendars/tests/CalendarHelpers.cs

```diff
@@ -161,49 +164,76 @@ public static IEnumerable<object[]> DateTime_TestData()
         }
 
         [Theory]
-        [MemberData(nameof(Year_Month_Day_Era_TestData), DataType.Year)]
+        [MemberData(nameof(Year_Month_Day_Era_TestData), DataType.Year, false)]
         public static void GetDaysInYear_Invalid(Calendar calendar, int year, int month, int day, int era, string exceptionParamName)
         {
             Assert.Throws<ArgumentOutOfRangeException>(exceptionParamName, () => calendar.GetDaysInYear(year, era));
         }
 
         [Theory]
-        [MemberData(nameof(Year_Month_Day_Era_TestData), DataType.Year)]
+        [MemberData(nameof(Year_Month_Day_Era_TestData), DataType.Year, false)]
```

Comment:
This test and the one above pass isdesktoprun=false even though they run on all TFM's. If it just means "this parameter is not relevant to this test" then possibly it would be clearer to make it a nullable bool and pass null? I don't feel strongly.